.github/workflows/piro-test-main.yml

```diff
@@ -1,6 +1,9 @@
 name: Testing - main
 
-on: [push]
+on:
+  push:
+    paths:
+      - 'piro/**/*'
```

Comment:
I think `web/backend` should be added.

Reply:
@ChrisFajardo-TRI please review if this is correct.

Reply:
no need for web/backend. we don't have tests for that, the tests are for the Piro folder